### PR TITLE
Django 2.2.x upgrade

### PR DESCRIPTION
**Pull request by movestill**
Upgrade to Django 2.2.13.

* Removed abandoned `django-oidc` dependency in favor of `mozilla-django-oidc`
* Removed use of our fork of `drf-oidc-auth` now that the original repo uses thread locking
* Redirect to login page after logging out

Related PRs:
https://github.com/jhuapl-boss/boss-tools/pull/37
https://github.com/jhuapl-boss/boss-manage/pull/82

### CODE REVIEW

File: django/boss/settings/production.py [hunk 1]
```diff
@@ -57,25 +58,42 @@
...
+OIDC_OP_AUTHORIZATION_ENDPOINT = auth_uri + '/protocol/openid-connect/auth'
+OIDC_OP_TOKEN_ENDPOINT = auth_uri + '/protocol/openid-connect/token'
+OIDC_OP_USER_ENDPOINT = auth_uri + '/protocol/openid-connect/userinfo'
+LOGIN_REDIRECT_URL = public_uri + 'v1/mgmt'
+LOGOUT_REDIRECT_URL = auth_uri + '/protocol/openid-connect/logout?redirect_uri=' + public_uri
+OIDC_RP_CLIENT_ID = client_id
+OIDC_RP_CLIENT_SECRET = ''
+OIDC_RP_SCOPES = 'sub preferred_username'
+OIDC_RP_SIGN_ALGO = 'RS256'
+OIDC_OP_JWKS_ENDPOINT = auth_uri + '/protocol/openid-connect/certs'
```
Comment (sandyhider) [inline django/boss/settings/production.py:91]: So how do you figure out you needed to add all these OIDC values?  Are they from mozilla_django_oidc?

File: django/boss/settings/base.py [hunk 2]
```diff
@@ -69,16 +69,19 @@
     'bootstrapform', # style management console
     'bossingest',
     'bossobject',
-    'rest_framework_swagger',
+    #'rest_framework_swagger',
```
[the remark this answers is not in the document]
Comment (sandyhider) [reply django/boss/settings/base.py:72]: I guess I see the new swagger under urls.py. /docs

File: django/bosscore/urls/resource_urls.py [hunk 3]
```diff
@@ -15,6 +15,7 @@
 from django.conf.urls import url
 from bosscore.views import views_resource
 
+app_name = 'bosscore'
```
Comment (sandyhider) [inline django/bosscore/urls/resource_urls.py:18]: is app_name a new variable in Django?

Comment (movestill) [reply django/bosscore/urls/resource_urls.py:18]: IIRC yes it’s now required.

File: django/boss/settings/production.py (hunk 1 above)
Comment (movestill) [reply django/boss/settings/production.py:91]: The docs and experimentation. I had to run everything locally to figure this out.

**Pull request merged**